ARE/components/processor.comport/src/main/java/eu/asterics/component/processor/comport/ComPortInstance.java

```diff
@@ -198,6 +211,24 @@ public Object setRuntimePropertyValue(String propertyName, Object newValue) {
         return null;
     }
 
+    final IRuntimeEventListenerPort elpEnablePlugin = new IRuntimeEventListenerPort() {
+        public void receiveEvent(final String data) {
+            if (!propEnabled) {
+                propEnabled = true;
+                start();
+            }
+        }
+    };
+
+    final IRuntimeEventListenerPort elpDisablePlugin = new IRuntimeEventListenerPort() {
+        public void receiveEvent(final String data) {
+            if (propEnabled) {
```

Comment:
the same as above

Reply:
here it would be possible to call `stop()` first, but it wouldn't make any difference to my mind and it looks somewhat inconsistent with the order in `elpEnablePlugin`.